composer.json

```diff
@@ -252,12 +252,6 @@
             "Tests\\": "tests/"
         }
     },
-    "repositories": [
-        {
-            "type": "vcs",
-            "url": "https://github.com/PrestaShop/jwt"
-        }
-    ],
```

Comment:
It may be interesting to archive the repo

Reply:
As soon as this is merged, please ping a GitHub administrators to do it ;) like me, @jolelievre, @kpodemski ...